src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  let price = 0;
+
+  if (days < 3) {
```

Comment:
Since you are using `if` and `else if` statements, there is no need to check for `days >= 3`, as it's already implied by the previous condition. 😉

Reply:
Code is rewrited